src/main/java/service/LineParserImpl.java

```diff
@@ -0,0 +1,16 @@
+package service;
+
+import shop.Fruit;
+import shop.FruitShopOperation;
+
+public class LineParserImpl implements LineParser {
+
+    @Override
+    public FruitShopOperation parseLine(String inputLine) {
+        String[] arrayLines = inputLine.split(",");
```

Comment:
```suggestion
        String[] arrayLines = inputLine.split(SEPARATOR);
```
краще зробити цю константу публічною і реюзати її так код буде більш читабельним